### PR TITLE
Lession 01: BASIC

### PR DESCRIPTION
**Pull request by hatan**
@kamwar lesson 01, second attempt  :)

### CODE REVIEW

File: students/kosarzewski_maciej/lesson_01_basics/fractional_part.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Read a float:
+a = float(input())
+# Print a value
+print (a - int(a))
```
Comment (kamwar) [inline students/kosarzewski_maciej/lesson_01_basics/fractional_part.py]: PEP8 warning

Please check PEP8 warnings with 'Code' -> 'Inspect code'

File: students/kosarzewski_maciej/lesson_01_basics/total_cost.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Read an integer:
+A = int(input())
```
Comment (kamwar) [inline students/kosarzewski_maciej/lesson_01_basics/total_cost.py]: Capital letters should be used only for constants e.g.
MY_NAME = "Kamil"

Please use lower case with underscores for variable names instead:

File: students/kosarzewski_maciej/lesson_00_the_setup/importable_module.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
```
Comment (kamwar) [inline students/kosarzewski_maciej/lesson_00_the_setup/importable_module.py]: Lesson 00 should not be a part of this PR. Lesson 00 is already delivered into jedzej/tietopythontraining-basic

File: students/kosarzewski_maciej/lesson_01_basics/fractional_part.py (hunk 1 above)
Comment (hatan) [reply students/kosarzewski_maciej/lesson_01_basics/fractional_part.py]: For my PyCharm is ok :\

Comment (kamwar) [reply students/kosarzewski_maciej/lesson_01_basics/fractional_part.py]: There should be no space after print statement. For some reason your pyCharm doesn't check PEP8. This time it's ok but try to follow PEP8 style guide next time.

**Pull request merged**